### DISCUSSION TITLE
Joining contributor team

### DISCUSSION THREAD

**Discussion by JohannesBeranek** [playframework/play1#1453] [General]
I'd be happy joining the contributor team if possible, as we (our dev team) has very recently forked play due to a handful of bugs and issues. Issue with just "opening a PR" is that we use play for our company software, so we can't just wait till someone comes around and pushes the merge button.

We don't plan to add/change anything backwards incompatible, but there's a couple of bugs I'd like to fix (like the restart loop in dev mode that you get about once an hour if you do development the whole day, or the properties enhancer breaking some hot reloading scenarios) as well as updating some of the core dependencies like netty (@tazmaniax I saw that you have a netty 4 branch open, and that netty 5 is currently being released - would be nice if we could upgrade play to netty 5).

**Comment by asolntsev** [comment]
@xael-fry

**Comment by aleksandy** [comment]
The main problem is that no one de facto needs play1. And first of all to its developers. In another way, I can't explain why even the simplest PR correcting obvious mistakes can wait months for a merge.

I advise you to think about migrating to more actively developed projects. For example, [replay](https://github.com/codeborne/replay), [micronaut](https://micronaut.io/), [quarkus](https://quarkus.io/), [klite](https://github.com/codeborne/klite) (if you like kotlin).

@xael-fry is too inactive on github, so don't expect his answer anytime soon.

**Comment by xael-fry** [comment]
I add @JohannesBeranek and @tazmaniax as contributor on play1.
Have fun

**Reply by mkurz** [reply]
Hi @JohannesBeranek and @tazmaniax,

I am the admin of the whole [GitHub playframework organization](https://github.com/playframework/). I just invited you to the "Play 1 Integrators" group where all maintaines of Play 1 are put into. Please accept my invitation, otherwise I will remove you as maintainer from this repo in a couple of days. This is just about organising members, for you nothing will change.

@xael-fry Next time you add new maintainers, please invite them to [this team](https://github.com/orgs/playframework/teams/play-1-integrators/members), I just gave you maintainer rights on this team so you can add and remove people. Currently you added the people as "outside collaborators" which is not ideal, I prefer to manage people in teams, so I can change permission in bulks etc.

Thanks!

**Reply by mkurz** [reply]
@JohannesBeranek please accept the invitation thanks!

**Reply by mkurz** [reply]
@JohannesBeranek you still did not accept the invitation. If you are still interested, let me know, I have to re-send it.

**Comment by jvosloo** [comment]
This is good news. We run a large commercial Fintech app on Play1. Migrating off of Play1 to something else would be a big undertaking. We're very thankful for the folks keeping the framework working with the latest versions of Java, upgrading the dependencies, fixing bugs, etc. 

We've just upgraded our code base from Java 8, Play1.5.3 to Java 17 and Play 1.7.1 a week ago and besides some trouble with module dependencies, it was a very smooth upgrade. 

It's therefore great to see some more active maintainers joint the team! 🙏🙌🥂

**Comment by aleksandy** [comment]
@xael-fry, could you add me into team too?
